projects/core/components/textfield/textfield-multi/textfield-multi-item.directive.ts

```diff
@@ -0,0 +1,18 @@
+import {Directive} from '@angular/core';
+import type {PolymorpheusContext} from '@taiga-ui/polymorpheus/classes/context';
+
+@Directive({
+    standalone: true,
+    selector: 'ng-template[tuiTextfieldItem]',
+})
+export class TuiTextfieldItem {
```

Comment:
Do we really need this one? 🤔 Maybe we can use `*tuiItem`? I wonder if types are worth extra cognitive load of a separate entity.